htdocs/admin/modulehelp.php

```diff
@@ -348,7 +348,7 @@
     $text.='<br><br>';
 
     $text.='<br><strong>'.$langs->trans("AddDataTables").':</strong> ';
-	$sqlfiles = dol_dir_list(dol_buildpath($moduledir.'/sql/'), 'files', 0, 'llx.*\.sql', array('\.key\.sql'));
+	$sqlfiles = dol_dir_list($moduledir.'/sql/', 'files', 0, 'llx.*\.sql', array('\.key\.sql'));
```

Comment:
Removing dol_buildpath will make the feature not working for external module ?
Why removing method ?